audible.py

```diff
@@ -333,10 +333,22 @@ def get_album_info(self, asin):
         data_url = f"https://api.audnex.us/books/{asin}"
 
         self.cover_art_urls[asin] = cover_url
+
+        original_year=year
+        original_month=month
+        original_day=day
+
+        if self.config['goodreads_apikey']:
+            original_date = search_goodreads(asin, self.config['goodreads_apikey'])
+            if original_date["year"]:
```

Comment:
This errors if `year` is not present. Did you mean to use dict.get?

Reply:
Using get for this code now.